lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -359,6 +359,7 @@ public function render_migration_ws( $args = array() ) {
 				$this->get_dashboard_link()
 			);
 			$this->render_field_description( 'Security token:' );
+			$this->render_const_notice( 'migration_token' );
```

Comment:
Look 2 lines below :)

Reply:
Wait so, this field was never allowed to be changed but now it is through constants? Isn't that wrong?

Reply:
It's a token to allow an Auth0 rule to authenticate with the database. You could never edit it in the admin but you can, now, set it in a constant. It will be part of the documentation to mention that changing this will require the rule to be changed as well.